pattern-2/bosh-release/README.md

```diff
@@ -1,93 +1,249 @@
 # BOSH release for WSO2 Identity Server deployment pattern 2
 
-This repository includes a BOSH release that can be used to deploy WSO2 Identity Server 5.4.0 deployment pattern 2
-configured to use a MySQL database on BOSH Director.
+This directory contains the BOSH release implementation for WSO2 Identity Server 5.4.0
```

Comment:
Shall we also change this according to the discussed action items with pattern 1 documentation in https://github.com/wso2/pivotal-cf-is/pull/35 ?

Reply:
This will be fixed in a future PR.